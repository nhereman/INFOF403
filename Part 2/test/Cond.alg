co Parser Test File ExprArith co

begin
	read(a);
	read(b);
	read(c);
	if a = b and c + 1 /= 2 or not c/2 = 4 then
		print(a)
	else
		print(c)
	fi;
	print(b) 
end