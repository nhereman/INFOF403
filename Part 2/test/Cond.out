1
3
4
12
49
6
4
12
49
6
4
12
49
6
4
8
28
31
34
37
39
14
17
22
19
16
40
14
17
22
19
16
35
37
39
14
17
22
19
15
24
17
23
19
16
45
14
17
23
19
16
36
32
34
38
39
14
17
22
18
27
23
19
16
40
14
17
23
19
16
36
33
3
4
11
48
5
30
3
4
11
48
5
6
4
11
48
5
